src/components/equations/ItemEffects.ts

```diff
@@ -98,4 +98,20 @@ export class ItemEffects<TGameStartr extends FullScreenPokemon> extends GeneralC
             });
         this.gameStarter.menuGrapher.setActiveMenu("GeneralText");
     }
+
+    /**
+     * Allows you to add a wild Pokemon to your party.
+     *
+     * @param pokemon   A Pokemon that you're capturing.
+     * @returns Whether the Pokemon was added to your party or not.
+     */
+    public capturePokemon(pokemon: IPokemon): boolean {
+        if (this.gameStarter.itemsHolder.getItem(this.gameStarter.storage.names.pokemonInParty).length >= 6) {
+            return false;
+        }
+        this.gameStarter.itemsHolder.setItem(this.gameStarter.storage.names.pokemonInParty, [
```

Comment:
This replaces the original array with a new one containing just the new Pokemon. I think you want to push to the original instead.